presto-main/src/test/java/com/facebook/presto/execution/TestQueryInfo.java

```diff
@@ -155,6 +155,7 @@ private static QueryInfo createQueryInfo()
                 URI.create("1"),
                 ImmutableList.of("number"),
                 "SELECT 1",
+                Optional.of("SELECT 1"),
```

Comment:
This is confusing, lets replace this with "template_query"

Reply:
Updated accordingly.